<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtInterfaceClass</class>
 <widget class="QMainWindow" name="QtInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtInterface</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: #0f0f0f;
	color: #fff;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;
	font-weight: bold;
	border-color: #000000;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #d10000;
	color: #fff;
	font-weight: bold;
	border: 1px solid #d10000;  
	border-radius: 2px;
	padding: 5px;

}


QPushButton::hover
{
	background-color: #ad0000;
	color: #fff;

}


QPushButton::pressed
{
	background-color: #e00000;
	color: #fff;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: transparent;
	color: #000000;
	border-style: solid;
	border-color: #000000;

}


QToolButton::hover
{
	background-color: #d10000;
	color: #000000;
	border-radius: 15px;
	border-color: #d10000;

}


QToolButton::pressed
{
	background-color: #d10000;
	color: #000000;
	border-style: solid;
	border-width: 0px;

}


/*-----QLineEdit-----*/
QLineEdit{
	background-color: #4d4d4d;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-radius: 5px;
	padding: 5px;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #b9b9bb;
	font-weight: bold;

}


QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #00111d;
    border: 1px solid #d10000;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;); /*To replace*/
	background-color: #d10000;
    border: 1px solid #d10000;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #ff0000;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}
</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>251</width>
       <height>239</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="6" column="1">
       <widget class="QPushButton" name="btn_log_in">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Log in</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>XO Oriel</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLineEdit" name="le_password">
        <property name="toolTip">
         <string/>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="maxLength">
         <number>32767</number>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btn_sign_up">
        <property name="text">
         <string>Sign up</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>XO Oriel</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Server IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="le_server_ip"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLineEdit" name="le_login"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>XO Oriel</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtInterface.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_sign_up</sender>
   <signal>clicked()</signal>
   <receiver>QtInterfaceClass</receiver>
   <slot>sign_up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_log_in</sender>
   <signal>clicked()</signal>
   <receiver>QtInterfaceClass</receiver>
   <slot>log_in()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sign_up()</slot>
  <slot>log_in()</slot>
  <slot>readyRead()</slot>
 </slots>
</ui>
